tweepy/asynchronous/streaming.py

```diff
@@ -66,8 +68,17 @@ def __init__(self, consumer_key, consumer_secret, access_token,
             f"Tweepy/{tweepy.__version__}"
         )
 
-    async def _connect(self, method, endpoint, params={}, headers=None,
```

Comment:
This is not related to type hints but this default value `params={}` can be potentially a cause of future bugs. As you may know, default values of method arguments can be reused. Setting mutable objects can result in sharing the same objects among multiple method calls.